code/modules/projectiles/gun.dm

```diff
@@ -212,6 +214,7 @@
 				shoot_with_empty_chamber(user)
 				break
 			process_chamber()
+			process_laser(target, user, params)
```

Comment:
"laser" is ambiguous, since the var is ```pointer_icon``` (should really be ```pointer_icon_state```) then this should also be ```process_pointer()```